.appveyor.yml

```diff
@@ -1,11 +1,15 @@
 version: "{build}"
 clone_folder: c:\gopath\src\github.com\mackerelio\mkr
 environment:
+  VERSION: go1.14.1
   GOPATH: c:\gopath
-  GOROOT: c:\go112
+  GOROOT: c:\go
   GO111MODULE: on
 build: off
 install:
+  - rmdir %GOROOT% /s /q
+  - appveyor DownloadFile https://dl.google.com/go/%VERSION%.windows-amd64.zip
+  - 7z x %VERSION%.windows-amd64.zip -oC:\
```

Comment:
The official archive's top directory is **go/**, and *7z* cannot skip top directory. Thus we should remove **C:\go** then extract the archive into **C:\go**.

Reply:
It's acceptable, but I think `GOROOT` can be set to somewhere else, like `/opt/xxx` in *nix.

Reply:
There is C:\tools directory by default. GOROOT was moved to it.